.github/workflows/verify-rulesets.yml

```
@@ -0,0 +1,94 @@
+      - '.github/workflows/verify-rulesets.yml'
+  push:
+    branches: [master]
+    paths:
+      - '.github/rulesets/**'
+      - '.github/workflows/verify-rulesets.yml'
+  schedule:
+    - cron: '0 8 * * *'   # daily at 08:00 UTC, catches out-of-band UI edits
+  workflow_dispatch:
+
+permissions:
+  administration: read
```

actionlint (1.7.12)

[error] unknown permission scope "administration". all available permission scopes are "actions", "artifact-metadata", "attestations", "checks", "contents", "deployments", "discussions", "id-token", "issues", "models", "packages", "pages", "pull-requests", "repository-projects", "security-events", "statuses"

(permissions)